src/Components/Details.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+const Details = () => {
+  const details = useSelector((state) => state.coins.selectedCoin);
+
+  return (
+    <div data-testid="Details-1" className="flex flex-col gap-2 justify-start items-center mt-10">
+      <img className="h-20" src={details.icon} alt={details.name} />
+      <h2>
+        Name :
+        {details.name}
+      </h2>
+      <h4>
+        Rank :
+        {details.rank}
+      </h4>
+      <h4>
+        Total Supply :
+        {details.totalSupply}
+      </h4>
+      <h4>
+        Available Supply :
+        {details.availableSupply}
+      </h4>
+      <h4>
+        Price :
+        {details.price}
+      </h4>
+      <h4>
+        Price Change Weekly :
+        {details.priceChange1w < 0 ? <span className="text-red-400">{details.priceChange1w}</span> : <span className="text-green-400">{details.priceChange1w}</span>}
+      </h4>
+      <h4>
+        Price Change Daily :
+        {details.priceChange1d < 0 ? <span className="text-red-400">{details.priceChange1d}</span> : <span className="text-green-400">{details.priceChange1d}</span>}
+      </h4>
+      <h4>
+        Price Change Hourly :
+        {details.priceChange1h < 0 ? <span className="text-red-400">{details.priceChange1h}</span> : <span className="text-green-400">{details.priceChange1h}</span>}
+      </h4>
+    </div>
```

Comment:
Nice and beuatiful design so far, however, as stated in the project requirements, you are suppose to follow the design guidelines including: 

- [ ] Colors (select one main color for your website).
- [ ] Typography: font face, size and weight.
- [ ] Layout: composition and space between elements.

There are some differences between yours and that of the actual design which you will need to fix.

1. The background of each odd card is darker than its preceeding (even) card forming a checkered layout.
2. Also, kindly note the margin, paddings, and distance between the cities name and the views
3. Your design is also missing the small arrow to the right of each card. 

### Your design 

![Screenshot from 2023-02-07 18-49-08](https://user-images.githubusercontent.com/65233860/217341976-cc71db77-56bb-4c0f-8afd-2f5b38086b63.png)


### The actual design 

![Screenshot from 2023-02-07 18-50-02](https://user-images.githubusercontent.com/65233860/217342011-f5bc0abd-fc26-4616-bfa8-9c25942c72c9.png)

Reply:
Thank you for your review @jerryowusu I have done my best to implement the requested style changes.

![Screenshot 2023-02-08 020622](https://user-images.githubusercontent.com/114587346/217359595-17846bc4-23b8-4f7a-b044-55bb929a9f5d.png)